sync/syncer.go

```diff
@@ -69,7 +79,7 @@ const (
 )
 
 func (s *Syncer) IsSynced() bool {
-	return s.VerifiedLayer() == s.maxSyncLayer()
+	return s.VerifiedLayer() == s.maxSyncLayer()-1
```

Comment:
I prefer incrementing to avoid int undeflow when maxSyncLayer = 0 
`return s.VerifiedLayer() +1 == s.maxSyncLayer()`

Reply:
also, verified layer is the last layer verified by tortoise, this is not a good sync indicator as it does not include the latest layer blocks.  I've had bugs with this, my solution was to use the last seen layer id instead, also, you've changed the sync criteria for the entire system, maybe we should add another function with this criteria

Reply:
that was a bit misleading its the last layer we pushed to tortoise( i fixed the name)
the last seen layer was not used so i removed it